extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.resteasy.test;
+
+import static org.hamcrest.Matchers.is;
+
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusDevModeTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+public class NotFoundExceptionMapperTestCase {
+    @RegisterExtension
+    static QuarkusDevModeTest test = new QuarkusDevModeTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(RootResource.class));
+
+    @Test
+    public void testHtmlResourceNotFound() {
+        // test the default exception mapper provided in dev mode : displays HTML by default
+        RestAssured.when().get("/not_found")
+                .then()
+                .statusCode(404)
+                .body(Matchers.containsString("<div class=\"component-name\"><h1>Resource Not Found</h1>" +
+                        "<h2>REST interface overview</h2></div><h2>/</h2>"));
```

Comment:
Oh wait, another comment here. Let's just test that it contains `<h1>Resource Not Found</h1>`.

It will be more resistant to potential future changes to the HTML output.

Reply:
done